experiments/run_utility.py

```diff
@@ -62,16 +62,16 @@ def run_ooo(aggregators, functions, degrees, name_base, sample_size=5):
                     if d > w:
                         continue
                     row = [f, w, d]
-                    print w, d, ':',
+                    print(w, d, ':',)
```

Comment:
We might need `print(w, d, ':', end='')` to prevent it from starting a new line; a trailing comma at the end won't do it in Python 3.